eng/pipelines/build.yml

```diff
@@ -163,34 +166,83 @@ stages:
         - Build_and_Test
         - Release
       jobs:
-        - job: Update
-          timeoutInMinutes: 180
-          pool:
-            vmImage: ubuntu-20.04
+        - template: update-generator-versions.yml
+# update codes
+        - template: update-azure-sdk-for-net-codes.yml
+          parameters:
```

Comment:
I'm curious about how we determined the filter. Is it determined by the number of RPs under different filter? For example, the number of RPs that the name starts with `a,b,c` is equal to the RPs that starts with `d,e`.

Reply:
> I'm curious about how we determined the filter. Is it determined by the number of RPs under different filter? For example, the number of RPs that the name starts with `a,b,c` is equal to the RPs that starts with `d,e`.

It's just a guess. I actually tuned the division once. We can further optimize.

Reply:
Got it, thanks, I think we can do something like [here](https://github.com/Azure/azure-sdk-for-net/blob/main/eng/scripts/splittestdependencies/generate-dependency-functions.ps1#L4) in the future to split the projects under SDK directory and dynamically generate the corresponding jobs accordingly.

Reply:
@Yao725 file https://github.com/Azure/autorest.csharp/issues/3165 to track